public/cli/lib/rendering/recordFrames.spec.ts

```diff
@@ -0,0 +1,28 @@
+import test from 'ava';
+import tempy from 'tempy';
+import fs from 'fs';
+import { record } from './recordFrames';
+// TODO: Add a puppeteer mock model
+test('recordFrames: render 5 mock frames', async (t) => {
+  let htmlContent = createMockHtml();
```

Comment:
If the value doesn't change use `const`.

Reply:
done.